sdk/rust/oak/src/lib.rs

```diff
@@ -191,10 +200,10 @@ pub fn channel_write(half: WriteHandle, buf: &[u8], handles: &[Handle]) -> OakSt
 /// On success, returns [`WriteHandle`] and a [`ReadHandle`] values for the
 /// write and read halves (respectively).
 pub fn channel_create() -> Result<(WriteHandle, ReadHandle), OakStatus> {
-    let mut write = WriteHandle { handle: 0 };
-    let mut read = ReadHandle { handle: 0 };
+    let mut write = WriteHandle { handle: Handle { id: 0 } };
```

Comment:
Perhaps define a `Handle::invalid()` method (and corresponding `INVALID_HANDLE` const)?